### PR TITLE
Only return surrogate pk for schemas with compound pk if schema does not define `id`

### PR DESCRIPTION
**Pull request by DonQueso89**
`get_fields()` now returns `id` twice for for example the verblijfsobjecten dataset

### CODE REVIEW

**Comment by lbam** [comment]
Test?

**Comment by lbam** [comment]
Nice, but I think you forgot to check in the `compound_key.json`?

**Comment by DonQueso89** [comment]
> Nice, but I think you forgot to check in the `compound_key.json`?

whops

File: tests/files/compound_key.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      "type": "table",
+      "version": "0.0.1",
+      "temporal": {
+        "identifier": "volgnummer",
+        "dimensions": {
+          "geldigOp": ["beginGeldigheid", "eindGeldigheid"]
+        }
+      },
+      "schema": {
+        "$id": "https://github.com/Amsterdam/schemas/afracnswallows",
```
Comment (lbam) [inline tests/files/compound_key.json:20]:
```suggestion
        "$id": "https://github.com/Amsterdam/schemas/africanswallows",
```

**Review by lbam — APPROVED** [review]

**Pull request merged**